docs/src/getting-started/override.rst

```diff
@@ -0,0 +1,82 @@
+Override Architecture's Default Parameters
+==========================================
+
+In our initial tutorial, we used default parameters to train a model employing the
+SOAP-BPNN architecture, as shown in the following config:
+
+.. literalinclude:: ../../static/options.yaml
+   :language: yaml
+
+While default parameters often serve as a good starting point, depending on your
+training target and dataset, it might be necessary to adjust the architecture's
+parameters.
+
+First, familiarize yourself with the specific parameters of the architecture you intend
+to use. We provide a list of all architectures and their parameters in the
+:ref:`available-architectures` section. For SOAP-BPNN models, the parameters are
+detailed at :ref:`architecture-soap-bpnn`.
+
+Modifying Parameters (yaml)
+---------------------------
+
+As an example, let's increase the number of epochs (``num_epochs``) and the ``cutoff``
+radius of the SOAP descriptor. To do this, create a new section in the ``options.yaml``
+named ``architecture``. Within this section, you can override the architecture's
+hyperparameters. The adjustments for ``num_epochs`` and ``cutoff`` look like this:
+
+.. code-block:: yaml
+
+   defaults:
+      - architecture: soap_bpnn
+      - _self_
+
+   architecture:
+      model:
+         soap:
+            cutoff: 7.0
+      training:
+         num_epochs: 200
+
+   # Dataset config omitted for simplicity.
+
+Modifying Parameters (Command Line Overrides)
+---------------------------------------------
+
+For quick adjustments, command-line overrides are also an option. The changes above can
+be achieved by:
+
+.. code-block:: bash
+
+   metatensor-models train options.yaml \
+      -y architecture.model.soap.cutoff=7.0 architecture.training.num_epochs=200
+
+Here, the ``-y`` flag is used to parse the override flags. More details on override
+syntax are available at https://hydra.cc/docs/advanced/override_grammar/basic/.
+
+.. note::
+   `metatensor-models` always writes the fully expanded ``options.yaml`` for your
```

Comment:
we should tell where the fully expanded file can be found

Reply:
Yes, good point.